ingress/controllers/nginx/nginx/main.go

```diff
@@ -67,7 +67,9 @@ func NewManager(kubeClient *client.Client) *Manager {
 
 	ngx.sslDHParam = ngx.SearchDHParamFile(config.SSLDirectory)
 
-	ngx.loadTemplate()
+	if err := ngx.loadTemplate(); err != nil {
+		glog.Fatalf("invalid NGINX template: %v", err)
```

Comment:
do you really want to fatal when a load fails? isn't this different from what nginx usually does? meanign if you have a working config and you overwrite it with some invalid config, nginx will keep working with the old config right?